course-outline.md

```diff
@@ -0,0 +1,99 @@
+## Course Outline: Introduction to Version Control 
+
+### Pre-Workshop Installation and Set Up
+
+Learning Objective: Set up a github account and install git (pre-workshop). \
+Instructions: [Installation and Set-Up](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/pre-workshop/00-software-install.html) \
+Assignment: [Pre-Workshop Assignment](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/pre-workshop/01-assignment.html) \
+
+Topics/Learning Outcomes:
+
+- Download Git for Windows and bundled software (includes Git Bash)
+- Create a GitHub account with m2FA, PAT for https connection
+- Configure git
+- Configure RStudio (optional)
+- Pre-workshop assignment to show local configuration and connection to GitHub
+
+Course Reference Material:
+
+- Installation instructions here: `glue::glue(safepaths::get_network_path(), "/8. Data Science Training/github_how_to")`
+- [SW Carpentry Episode 2. Setting Up Git](https://swcarpentry.github.io/git-novice/)
+- [bcgov-r-geo-workshop Lesson 1](https://github.com/bcgov/bcgov-r-geo-workshop/blob/master/20191105_Day_1_PM_Intro/lessons/01-setup.md)
+
+### Section 1. Introductions and What is Version Control?
+
+Learning Objective: Understand the value-add of using version control and the the basics of how automated version control systems work. \
+Slides: [Section 1 Slides](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/slides/01-what-is-vc/01-what-is-vc.html)
+
+Topics:
+
+- How version control systems work
+- What is a repository?
+- What is the value-add of using version control in teams? individually?
+
+Course Reference Material:
+
+- [SW Carpentry Episode 1. Automated Version Control](https://swcarpentry.github.io/git-novice/01-basics.html)
+
+### Section 2. Working local with git
+
+Learning Objective: Use command-line git to track local file changes. (*more about git*) \
+Slides: [Section 2 Slides](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/slides/02-working-local/02-working-local.html)
+
+Topics:
```

Comment:
I would be tempted to specifically include the topic of working areas vs staging areas vs the repository.